Src/Plugins/Security/SEC101_051.GpgCredentialValidator.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Utilities;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Internal
```

Comment:
>Internal [](start = 71, length = 8)

remove internal

Reply:
Done.

---
In reply to: [585942246](https://github.com/microsoft/sarif-pattern-matcher/pull/249#discussion_r585942246) [](ancestors = 585942246)